rulebook.tex

```diff
@@ -2171,27 +2171,25 @@ \section{The Robotino System}
 still allowed, however, we strongly recommend using the Robotino 3 as
 the robot hardware platform from this year on.
 
-Any kind of sensors can be changed or added to	 the Robotino platform.
+Any kind of sensors can be changed or added to the Robotino platform.
 However, it is not possible to implement sensors that require
 modifications outside the Robotino area (e.g., Northstar, indoor GPS).
 There must be no changes to the controller or mechanical system.
 The robots peripherals must not exceed the maximum total height of
 \SI{1.1}{\metre} including the tower and the table on top. Additional
 hardware (sensors, computing equipment, etc.)  must be within a circle
-of a diameter of \SI{0.65}{\metre}  (for both, Robotino 2 and 3) or
-centered at the robot's rotational center-point. Additional hardware
-may only occupy up to $25\%$ of this additional \SI{0.14}{\metre}
-(Robotino 2) or \SI{0.10}{\metre} (Robotino 3) wide ring around the
+of a diameter of \SI{0.55}{\metre} or centered at the robot's rotational
+center-point. Additional hardware may only occupy up to $25\%$ of this 
+additional \SI{0.05}{\metre} wide ring around the
 robot. The only additional actuator allowed is one gripping device for
 workpieces which can be the original or a modified one. In the resting
-position and while driving it also must be within the same diameter of
-the other additional hardware.
-During interaction with a machine, the gripper may be extended to
-twice the robot diameter, provided that the direction of the moving
-axis is within the clamped angle of both backside corner-points of the
-handled MPS\@.
-The gripper is allowed to transport one workpiece at a time. And the
-gripper must release the workpiece for safety whenever the referee
+position and while driving it also must not exceed the robot diameter by
+more than this 5cm, including workpiece.
+In front of a machine and during a production process, the gripper may
+be extended up to 30cm. But it is only allowed to reach a maximum of 
+15cm into the machine area.
+The gripper is allowed to transport one workpiece at a time. And it
+must release the workpiece for safety whenever the referee
 wants to take out it. For example, if the referee presses the release
 button or key, the gripper releases the workpiece.
```

Comment:
Are we sure this is implemented yet?
The way this is formulated the RefBox has to have a function to request a specific robot to open the gripper. I have not seen this being implemented with any team as referee's in the past have forcefully taken out workpieces. For good conduct this should be implemented properly (including refbox message) or postponed for 2020.

Reply:
This was the original text before, i didn't  changed that. But of course it's not comprehensible because there's no mechanism to open the gripper. Should i remove this phrase?

Reply:
Please remove the phrase about opening the gripper. Then I agree with the text.